_data/vnext_faq.yml

```diff
@@ -133,7 +133,7 @@ consumers:
            The API ML in version 3 supports clients with onboarding enablers from version 2. It also works the other way around the onboarding enablers from version 3 will continue working with Zowe v2
 
        - number: 2
-         question: For us, as for extenders, jump to Java 17 means that we have to maintain 2 separate versions of our application - one with apiml-enabler v2 and another one with v3 to support customers who want to stay with Java 8. Do you have any recommendation or a workaround how to support both java versions?
+         question: For us, as for extenders, the jump to Java 17 means that we have to maintain 2 separate versions of our application - one with apiml-enabler v2 and another one with v3 to support customers who want to stay with Java 8. Do you have any recommendation or a workaround how to support both java versions?
```

Comment:
spell out "2"

also, consider this optional version:

For extenders, the jump to Java 17 means maintaining two separate versions of our application to support customers who want to stay with Java 8. Do you have a recommendation or a workaround for supporting both java versions? (should java be capitalized?)

(If you really need the "one with apiml-enabler v2 and another one with v3" bit, I'd include it as a parenthetical.)